packages/backend-core/src/queue/inMemoryQueue.ts

```diff
@@ -133,7 +136,14 @@ class InMemoryQueue implements Partial<Queue> {
    * a JSON message as this is required by Bull.
    * @param repeat serves no purpose for the import queue.
    */
-  async add(data: any, opts?: JobOptions) {
+  // add(name: string, data: T, opts?: JobOptions): Promise<Job<T>>;
+  async add(data: T | string, optsOrT?: JobOptions | T) {
+    if (typeof data === "string") {
+      throw new Error("doesn't support named jobs")
+    }
+
+    const opts = optsOrT as JobOptions
```

Comment:
These changes were required to do the correct generic typing on the class (`InMemoryQueue<T>` as opposed to just `InMemoryQueue`).